src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/RSAPkcs1X509SignatureGenerator.cs

```diff
@@ -47,7 +42,7 @@ internal static PublicKey BuildPublicKey(RSA rsa)
                 // This is due to one version of the ASN.1 not including OPTIONAL, and that was
                 // the version that got predominately implemented for RSA. Now it's convention.
                 new AsnEncodedData(oid, new byte[] { 0x05, 0x00 }),
-                new AsnEncodedData(oid, rsaPublicKey));
+                new AsnEncodedData(oid, rsa.ExportRSAPublicKey()));
```

Comment:
is this guaranteed to be DER?

Reply:
I don't think it's guaranteed to be DER, but it should be guaranteed to be valid ASN.1 BER representation that belongs in that place (RFC 8017).